.gitignore

```diff
@@ -5,3 +5,4 @@ results*
 *.json
 *.log
 tools/android-sdk-linux
+*.pid
```

Comment:
A PID file should be generated only to track the ipython server, right?
Thus, maybe better:
```bash
ipynb/*.pid
```